lib/iris/io/__init__.py

```diff
@@ -334,6 +334,13 @@ def save(source, target, saver=None, **kwargs):
         # Save a cube list to netCDF, using the NETCDF4_CLASSIC storage option
         iris.save(my_cube_list, "myfile.nc", netcdf_format="NETCDF3_CLASSIC")
 
+    .. warning::
```

Comment:
Fixed. I've rewritten the start and used much of your suggested text, which made sense to me.